### PR TITLE
Review Trial Projects

### PR DESCRIPTION
**Pull request by Wolfmikerson**
This pull request includes

1- Checked if I have used Semantic HTML tags
2- Checked if I have applied HTML & CSS best practices.
3- Checked if I had no linters error

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
-# Trial_Review
-A review on one of my Trial Project
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```
Comment (Wolfmikerson) [inline README.md:3]: Please, add a better description of your project here.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png
+      " alt="An image">
+      <nav id="nav-bar">
+        <button class="but" type="submit" name="submit">Element 1</button>
+        <button class="but" type="submit" name="submit">Element 2</button>
+        <button class="but" type="submit" name="submit">Element 3</button>
+      </nav>
+    </header>
+    <iframe id="video" width="420" height="315" src="https://www.youtube.com/watch?v=f9ip1iSAptQ">
+    </iframe>
```
Comment (Wolfmikerson) [inline index.html:22]: I think it is better if you close the tag frame in the above line.

**Review by Wolfmikerson — COMMENTED** [review]
Please see my review and make the appropriate changes.

File: README.md (hunk 1 above)
Comment (MahmoudBakr23) [reply README.md:3]: - [ ] This is a very polite way to suggest changes. However, I highly encourage that you should always consider telling why are you requesting these changes, some sort of an explanation to clarify. 😄 🙏

File: index.html (hunk 2 above)
Comment (MahmoudBakr23) [reply index.html:22]: - [ ] The same here as well, please provide an explanation for your request.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @Wolfmikerson 😃, 

### **Changes Required**

You managed to deliver a great and very polite review. However, there is always room for improvement 🙏 Kindly check my comments below your review's comments and update them with the changes 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Wolfmikerson) [inline index.html:22]: I think for better clarity, you should close the tag frame in the above line.

For example: 
<iframe id="id123" width="140" height="200" src="url"></iframe>

File: README.md (hunk 1 above)
Comment (Wolfmikerson) [inline README.md:3]: Please, add a better description of your project here. This will help the readers have a better understanding of the project.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Wolfmikerson 😃, 

Thank you for making the requested changes ☺️ Well done 👌🏻
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**